<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab2</class>
 <widget class="QWidget" name="Lab2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>64</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lab2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CezarCipher</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>AphineCipher</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cezar with keyword</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tricemus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
